products/ha_memgraph/v0.15.0/how_to_guides/query-ha-memgraph-programmatically.md

```diff
@@ -0,0 +1,222 @@
+## How to Query HA Memgraph Programmatically?
+
+If you have successfully
+[configured HA Memgraph](configure-and-run-ha-memgraph.md) along with the proxy
+setup, the process of programmatic querying should be exactly the same as
+it would be to query the single node Memgraph instance.
+
+### Supported Languages
+
+If users wish to query HA Memgraph programmatically, they can do so using the
+[Bolt protocol](https://boltprotocol.org). Bolt was designed for efficient
```

Comment:
...using the [Bolt protocol]... --> ...using the [Bolt protocol] proxy server...